Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -175,8 +178,28 @@
         return createNSArray(value, [] (double number) {
             return adoptNS([[NSNumber alloc] initWithDouble:number]);
         });
-    }, [] (const RetainPtr<NSTextAttachment>& value) -> RetainPtr<id> {
-        return value;
+    }, [] (const TextAttachmentMissingImage& value) -> RetainPtr<id> {
+        UNUSED_PARAM(value);
+        NSBundle *webCoreBundle = [NSBundle bundleWithIdentifier:@"com.apple.WebCore"];
+#if PLATFORM(IOS_FAMILY)
+        UIImage *missingImage = [PlatformImageClass imageNamed:@"missingImage" inBundle:webCoreBundle compatibleWithTraitCollection:nil];
```

Comment:
This is duplicate code.  Let's abstract this into a function that returns an image rather than duplicating it.